src/shared/components/palette/address/write-address-field.component.ts

```diff
@@ -74,8 +76,8 @@ export class WriteAddressFieldComponent extends AbstractFieldWriteComponent impl
   }
 
   focusAddressList() {
-    if (document.getElementById('addressList')) {
-      document.getElementById('addressList').focus();
+    if (document.getElementById(this.idFocus + '_addressList')) {
```

Comment:
we must remove document. 
it's not the recommended Angular way and can create other sort of issues. And it's causing unit tests to fails in a flaky way